docs/Encyclopedia/pages/SNP_Array-Based_Data.md

```diff
@@ -9,11 +9,10 @@ SNP Array-Based data is used at the GDC to perform Copy Number Variation analyse
 
 ### Data Formats ###
 
-The GDC stores processed SNP Array-Based data in the active harmonized portal. They are stored as tab-delimited copy number segmentation files, which identify the segment regions and their estimated copy number. The raw array intensity files in CEL format along with downstream copy number variation and genotyping analyses are available in the GDC Legacy Archive<sup>2</sup>.  
+The GDC stores processed SNP Array-Based data in the active harmonized portal. They are stored as tab-delimited copy number segmentation files, which identify the segment regions and their estimated copy number. 
```

Comment:
Same as above, the CEL files are available on active.  In terms of downstream copy number variation data, the "Birdseed" files are available.